ios/ApiPreferences.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ApiPreferences.swift
+//  magnet
+//
+//  Created by sam on 11/7/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class ApiPreferences: ApiBase {
+  
+  private static let PATH = "content://preferences"
+  private var preferences: Preferences!
```

Comment:
Oh! I though we were going to use the RequestStore, where we can store json directly, but tbh, I like this idea of having sqlite stores :)